packages/core/src/actions/contracts/readContracts.test.ts

```diff
@@ -1,3 +1,4 @@
+import { BigNumber } from 'ethers'
```

Comment:
On L48, the type of `await readContracts({ contracts })` resolves to `any[]`. Same goes for the `useContractReads` & `useContractInfiniteReads` tests. Should these be typed?